File: client/src/app/_models/eventParams.ts
import { User } from "./user";

export class EventParams {
    eventName: string;
    fromDate: string;
    toDate: string;
    country: string;
    region: string;
    city: string;
    eventTag: string;
    pageNumber = 1;
    pageSize = 10;

    constructor(user: User) {}
}

File: API/Helpers/Pagination/EventParams.cs
namespace API.Helpers
{
	public class EventParams : PaginationParams
	{
		public string? EventName { get; set; } 
		public string? FromDate { get; set; }
		public string? ToDate { get; set; }
		public string? Country { get; set; }
		public string? Region { get; set; }
		public string? City { get; set; }
		public string? EventTag { get; set; }
	}
}